website/api/getting-started.md

```diff
@@ -0,0 +1,61 @@
+---
+slug: /getting-started
+---
+
+# Getting Started
+
+To harness the power of ZK, you'll need to:
+
+1. [Write an application for the RISC Zero zkVM][zkvm-quickstart].
+2. [Generate proofs for your zkVM application][bonsai-quickstart].
+3. [Integrate your proofs into on-chain applications][foundry-template].
+
+```mermaid
+
+flowchart TD
+    %% Level 0
+    getting_started["Getting Started
+    To use the zkVM and/or Bonsai for blockchain applications, users will need to:
+    1. Build a zkVM app.
+    2. Generate proofs for their zkVM app.
+    3. Integrate proofs into blockchain applications."]
+
+
+
+    zkvm_app["1. zkVM Application Development"]
+        zkvm_quickstart["zkVM Quickstart"]
+    proof_generation["2. Generating proofs for your zkVM application."]
+
+    chain_integration["3. Blockchain Integration"]
+        getting_proofs_on_chain["Post proofs on-chain."]
+        onchain_verifier["Call our on-chain verifier"]
+
+
+    dev_mode["Dev Mode
+    - Skips proving enabling fast prototyping"]
+    local_proving["Local Proving
+    - Made possible by open-source tech
+    - Important for privacy applications"]
+    remote_proving["Remote Proving
+    - Bonsai is a scalable proving service, <br/>ready to generate proofs as needed."]
+
+
+
+    getting_started --> zkvm_app
+        zkvm_app --> zkvm_quickstart
+    getting_started --> proof_generation
+            proof_generation --> dev_mode
+            proof_generation --> local_proving
+            proof_generation --> remote_proving
+    getting_started --> chain_integration
+        chain_integration --> getting_proofs_on_chain
+        chain_integration --> onchain_verifier
+
+
+
+
+```
+
+[zkvm-quickstart]: ./zkvm/quickstart.md
+[bonsai-quickstart]: ./bonsai/quickstart.md
```

Comment:
`./bonsai/quickstart` appears to no longer be in the sidebar and is kind of stranded. it seems the remote proving page has all the information that used to be there, so linking there seems like the way to go, or simply to the "proving options" page. Either way, we should probably delete the Bonsai quickstart page.